.github/workflows/check-disclosures.yml

```diff
@@ -0,0 +1,33 @@
+name: Check disclosures 
+
+on: 
+  pull_request:
```

Comment:
`types: [assigned, opened, edited, synchronize, reopened]`

Reply:
This is likely unnecessary, as it defaults to `opened, synchronize, reopened`, and as this is only scanning the code (json files), we only need to run this when the code has changed (`opened` and `synchronize`)